API.md

```diff
@@ -355,43 +355,46 @@ public struct HTTPResponseStatus: Equatable, CustomStringConvertible, Expressibl
 }
 
 /// HTTP Methods handled by http_parser.[ch] supports
-public enum HTTPMethod : RawRepresentable {
-    /* be future-proof, http_parser can be upgraded */
-    case other(String)
+public struct HTTPMethod : RawRepresentable {
```

Comment:
May be `CustomStringConvertible` + `ExpressibleByStringLiteral` is a better choice? `RawRepresentable` is about interoperablity between Swift and C/Obj-C. I don't think it has any value here because usually underlying C implementation has C-enum for such task and doesn't accept string anyway.